content/international-candidates.md

```diff
@@ -52,217 +52,162 @@ keywords:
+### What you need to start teaching
 
-First, [choose your teaching training provider](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+To teach in England you need:
 
-Then, contact your chosen teacher training provider to check they are a Home Office licensed student sponsor – sponsors can help you apply for your visa.
+* teaching skills and experience
+* to pass criminal and professional safeguarding checks (these will be organised by your employer)
+* to [get the right visa or status](/international-candidates#get-the-right-visa-or-status)
 
-You’ll be able to apply for your visa if you:
+It is also helpful to have a teaching qualification from your own or another non-UK country, or to get '[qualified teacher status](/international-candidates#consider-getting-qualified-teacher-status)'.
 
-* have an unconditional offer of a place on a course with a Home Office licensed student sponsor teacher training provider
-* have a reference number, called a confirmation of acceptance for studies (CAS), which your training provider will give you
-* have enough money to support yourself and pay for your course
-* [can speak, read, write, and understand English](https://www.gov.uk/student-visa/knowledge-of-english)
+### Get the right visa or status
 
-### Applying for a skilled worker visa for a salaried teacher training course, or to work as a teacher
+You'll need to [get the right visa or status to work in the UK](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration).
 
-For a salaried teacher training course or to work as a teacher, you will need a [skilled worker visa](https://www.gov.uk/guidance/new-immigration-system-what-you-need-to-know#skilled-workers). Skilled worker visas are sponsored by Home Office licensed employers.
+Some job offers allow you to apply for a ‘skilled worker visa’. Otherwise, you may be eligible for another type of visa.
 
-### For salaried teacher training
+You may have a pre-existing right to work in the UK, such as settled or pre-settled status under the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families) or [indefinite leave to remain in the UK](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk).
 
-First, [choose your teaching training provider](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+[If you're an Irish citizen you do not need a visa](https://www.gov.uk/government/publications/common-travel-area-guidance).
 
-Then, contact your chosen teacher training provider to ask about the school where you’ll be employed during your training. The school must be a Home Office licensed employer sponsor – sponsors can help you apply for your visa.
+### Consider getting qualified teacher status
 
-### To work as a teacher
+You need 'qualified teacher status' (QTS) to teach in many state schools in England.
```

Comment:
Isn't it all state schools?

Reply:
i think you're right - will double check with policy